src/app/public/modules/phone-field/phone-field.component.scss

```diff
@@ -0,0 +1,41 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
+@import "~intl-tel-input/build/css/intlTelInput";
+
+
+.sky-input-group-country-btn {
+
+  ::ng-deep .sky-dropdown-button {
+    border-radius: 0;
+  }
+
+  ::ng-deep .sky-dropdown-item .iti-flag {
+    display: inline-block;
+    margin-right: 5px;
```

Comment:
Seems like a good candidate for skyux css vars? I'm guessing this 5px is probably based on the spacing of the parent dropdown and/or input field (which are all based on vars).

Reply:
Good catch. Fixed